data/publisher/src/main/java/tech/pegasys/teku/data/publisher/SystemMetricData.java

```diff
@@ -13,57 +13,220 @@
+
+  @JsonProperty("misc_node_boot_ts_seconds")
+  private final long miscNodeBootTsSeconds;
+
+  @JsonProperty("misc_os")
+  private final String miscOs = getNormalizedOSVersion();
+
+  public SystemMetricData(long timestamp, final File beaconNodeDirectory) {
+    super(timestamp, SYSTEM.getDisplayName());
+
+    SystemInfo systemInfo = new SystemInfo();
+    final HardwareAbstractionLayer hardware = systemInfo.getHardware();
+    hardware.getProcessor().getPhysicalProcessorCount();
+    this.cpuCores = hardware.getProcessor().getPhysicalProcessorCount();
+    this.cpuThreads = hardware.getProcessor().getLogicalProcessorCount();
+    this.memoryNodeBytesFree = hardware.getMemory().getAvailable();
+    this.memoryNodeBytesTotal = hardware.getMemory().getTotal();
+    this.miscNodeBootTsSeconds = getNodeBootSeconds();
+
+    final long[] processStats =
+        setProcessLoadTicks(hardware.getProcessor().getProcessorCpuLoadTicks());
+    this.cpuNodeUserSecondsTotal = processStats[0];
+    this.cpuNodeSystemSecondsTotal = processStats[1];
+    this.cpuNodeIdleSecondsTotal = processStats[2];
+    this.cpuNodeIowaitSecondsTotal = processStats[3];
+
+    this.networkNodeBytesTotalReceive =
+        hardware.getNetworkIFs(false).stream().map(NetworkIF::getBytesRecv).reduce(0L, Long::sum);
+    this.networkNodeBytesTotalTransmit =
+        hardware.getNetworkIFs(false).stream().map(NetworkIF::getBytesSent).reduce(0L, Long::sum);
+
+    this.diskNodeBytesTotal =
+        beaconNodeDirectory.exists() ? beaconNodeDirectory.getTotalSpace() : 0L;
+    this.diskNodeBytesFree = beaconNodeDirectory.exists() ? beaconNodeDirectory.getFreeSpace() : 0L;
+  }
+
+  private long getNodeBootSeconds() {
+    switch (miscOs) {
+      case "lin":
+        return new LinuxOperatingSystem().getSystemBootTime();
+      case "win":
+        return new WindowsOperatingSystem().getSystemBootTime();
+      case "mac":
+        return new MacOperatingSystem().getSystemBootTime();
+      default:
+        return 0L;
+    }
+  }
+
+  private long[] setProcessLoadTicks(final long[][] processorCpuLoadTicks) {
+
+    long idle = 0L;
+    long system = 0L;
+    long user = 0L;
+    long iowait = 0L;
+    for (long[] stats : processorCpuLoadTicks) {
+      user += stats[0];
+      system += stats[2];
+      idle += stats[3];
+      iowait += stats[4];
+    }
+    // load ticks are in millis, and we require seconds
+    user /= 1000;
+    system /= 1000;
+    idle /= 1000;
+    iowait /= 1000;
+    // note: currently beaconcha.in expects system time to be everything
+    return new long[] {user, (system + user + idle + iowait), idle, iowait};
+  }
+
+  private static String getNormalizedOSVersion() {
+    String currentVersionInfo = VersionProvider.VERSION;
+    if (currentVersionInfo.contains("linux")) return "lin";
+    if (currentVersionInfo.contains("windows")) return "win";
+    if (currentVersionInfo.contains("osx")) return "mac";
```

Comment:
I know this is nice and short but we always use braces on if statements.  Probably winds up working best as a series of if/else if